ui-frontend/packages/catalog-ui-search/src/main/webapp/component/filter-builder/filter-builder.tsx

```diff
@@ -29,30 +29,52 @@ type Props = {
 const getBaseFilter = ({ model }: { model: any }): FilterBuilderClass => {
   const filter = model.get('filterTree')
   if (filter.filters === undefined) {
-    return new FilterBuilderClass({
-      type: 'AND',
-      filters: [filter],
-      negated: false,
+    return validateFilter(
+      new FilterBuilderClass({
+        type: 'AND',
+        filters: [filter],
+        negated: false,
+      })
+    )
+  }
+  return validateFilter(
+    new FilterBuilderClass({
+      ...filter,
     })
+  )
+}
+
+/**
+ * Updates filter to accomodate WFS sources which don't handle blanks
+ */
+const validateFilter = (filter: FilterBuilderClass) => {
+  if (filter.filters.length === 1) {
+    if (
+      !isFilterBuilderClass(filter.filters[0]) &&
+      filter.filters[0].property === 'anyText' &&
+      filter.filters[0].type === 'ILIKE' &&
+      filter.filters[0].value === ''
+    ) {
+      filter.filters[0].value = '*'
```

Comment:
It looks like basic search is using "%" rather than "*".  Not sure why but probably should keep it consistent

Reply:
That transformation will happen during the normalization, so what we send over the wire is still %.